libs/shared/console-shared/src/lib/database-buttons-actions/ui/database-buttons-actions.tsx

```diff
@@ -0,0 +1,16 @@
+import { DatabaseEntity } from '@qovery/shared/interfaces'
+import { ButtonIconAction, ButtonIconActionElementProps } from '@qovery/shared/ui'
+
+export interface DatabaseButtonsActionsProps {
+  database: DatabaseEntity
+  environmentMode: string
+  buttonActionsDefault?: ButtonIconActionElementProps[] | undefined
+}
+
+export function DatabaseButtonsActions(props: DatabaseButtonsActionsProps) {
```

Comment:
Do you think we can harmonize the UIs with Database, Environment, Application ? Seems to be the sames